internal/process/processes.go

```diff
@@ -120,7 +120,7 @@ func CheckState(ctx context.Context, deployer deploy.Deployment, service deploy.
 }
 
 // Run executes the command
-func (a *actionWaitProcess) Run(ctx context.Context) (string, error) {
+func (a *actionWait) Run(ctx context.Context) (string, error) {
```

Comment:
I'm not an expert at go but we could also make `run` function private :)

Reply:
I'm checking the code, and there is an interface in the deploy package, `ServiceOperation`, that exposes a `Run` method. Therefore, this method should be kept public to match the interface.